modules/ROOT/pages/dw-core-functions-filter.adoc

```diff
@@ -3,85 +3,85 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-//* <<filter1>>
-//* <<filter2>>
-
 
 [[filter1]]
 == filter(Array<T>, (item: T, index: Number) -> Boolean): Array<T>
 
-Returns an array that containing elements that meet the criteria specified
-by a function (a lambda). The function is invoked with two parameters: `value`
-and `index`.
+Filters a list (array) leaving the items that meet the criteria specified by an expression.
+
```

Comment:
Extracts items (elements? are we consistent?) from an array that meet ...."

Reply:
Oh, I see the function actually just evaluates, it doesn't filter or extract, does it?

Reply:
Wait--I'm really confused. Is there a way to make this clearer?